client/matchmaking/find-match-forms.tsx

```diff
@@ -214,7 +216,7 @@ export function MapVetoesControl({
         ))}
       </MapSelections>
       <VetoStatus>
-        <VetoStatusLabel>Vetoes left: </VetoStatusLabel>
+        <VetoStatusLabel>{t('matchmaking.findMatch.vetoesLeft', 'Vetoes left:')} </VetoStatusLabel>
```

Comment:
This should probably be a `Trans` component with both of these parts in one string